src/engine/SCons/TaskmasterTests.py

```diff
@@ -935,7 +935,7 @@ def raise_UserError():
         except SCons.Errors.UserError:
             pass
         else:
-            raise TestFailed("did not catch expected UserError")
+            raise AssertionError("did not catch expected UserError")
```

Comment:
? why change assertion type?

Reply:
TestFailed is undefined.  This was a guess at something to replace it.

Reply:
... one assumes the else clause has never been touched, since we don't see this as an error.

Reply:
Looks like TestFailed() is a deprecated part of unittest.   I think the sugguested replacement would be ```self.fail()```

Reply:
ok, can change that if you like.

Reply:
Please do. Thanks!